src/ostree/ot-builtin-reset.c

```diff
@@ -33,7 +33,12 @@
  * man page (man/ostree-reset.xml) when changing the option list.
  */
 
+static gboolean opt_create;
+static char *opt_collection_id;
+
 static GOptionEntry options[] = {
+  { "create", 'c', 0, G_OPTION_ARG_NONE, &opt_create, "Create the ref if it doesn't exist already", NULL },
```

Comment:
```suggestion
  { "create", 'c', 0, G_OPTION_ARG_NONE, &opt_create, "Create the ref if it doesn’t exist already", NULL },
```

I’ll get software using curly quotes and apostrophes if it kills me.